### ISSUE TITLE
`allcaps that` deletes whitespace before highlighted string in Github 

### ISSUE THREAD

**Issue by xavier630** [knausj85/knausj_talon#550]
This happens in Github issues and PRs when editing both titles and descriptions.

I haven't done any investigation, I suppose it's likely to be some quirk of the Github editor related to something we insert.

Latest knausj develop, August 2nd 2021
Talon 0.2.0-8
Chrome 92.0.4515.107

**Comment by xavier630** [comment]
This also occurs for me with `allcaps that` and `title that` in gmail when writing emails.

**Comment by xavier630** [comment]
This also occurs for me in Outlook.

**Comment by splondike** [comment]
@xavier630 Hmm, I'm not able to reproduce this on Linux in Gmail or Github (see screenshot of Gmail for example). Are you still getting this issue?

![gmail-screenshot](https://user-images.githubusercontent.com/113718/136738643-424c1b75-2d77-4c5a-9fb7-674340821176.png)

**Comment by xavier630** [comment]
Yep this is what I get on github with the latest knausj (`allcaps that`):
![image](https://user-images.githubusercontent.com/9077250/136800358-c2d72dcb-64d6-4a88-bb50-659adefdf383.png)
![image](https://user-images.githubusercontent.com/9077250/136800362-8b214685-e26e-4e5a-8fbc-bff6e1ab4620.png)
